ibis/backends/polars/tests/test_client.py

```diff
@@ -22,3 +25,14 @@ def test_cannot_run_sql_after_drop(con):
     con.drop_table(name)
     with pytest.raises(PolarsSQLInterfaceError):
         con.sql(sql)
+
+
+def test_array_flatten(con):
```

Comment:
Moved this here in order to avoid dealing with Snowflake's PyArrow array output, which differs from other backends because of the untyped nature of its arrays.